jails.lua

```diff
@@ -23,9 +19,11 @@ function HandleJailCommand( Split, Player )
 			end
 			OtherPlayer:TeleportToCoords( jails[Tag]["x"] + 0.5 , jails[Tag]["y"] , jails[Tag]["z"] + 0.5)
 			OtherPlayer:SendMessageWarning('You have been jailed')
-			UsersIni:SetValue(OtherPlayer:GetName(),   "Jailed",   "true")
-			UsersIni:WriteFile("users.ini")
+			UsersINI:SetValue(OtherPlayer:GetName(),   "Jailed",   "true")
+			UsersINI:WriteFile("users.ini")
+			Jailed[OtherPlayer:GetName()] = true
 			Jailed = true
```

Comment:
But one is an array and the other a server variable.
Didnt know they could collide, but thanks, ill try later.